interface/usergroup/usergroup_admin.php

```diff
@@ -359,8 +359,8 @@
 ?>
 <html>
 <head>
-
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+<title><?php xl('User / Group', 'e');?></title>
+<link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'];?>/bootstrap-3-3-4/dist/css/bootstrap.css" type="text/css">
```

Comment:
for "bootstrap light light", to ensure compatible when got to full standard assets headers in future, shouldn't we:
1. include the bootstrap after include the $css_header
2. include the bootstrap rtl style if applicable